src/analysis/normed/group/pointwise.lean

```diff
@@ -48,46 +48,157 @@ begin
   ... ≤ Rs + Rt : add_le_add (hRs x hx) (hRt y hy)
 end
 
-@[simp] lemma singleton_add_ball (x y : E) (r : ℝ) :
-  {x} + ball y r = ball (x + y) r :=
-by simp only [preimage_add_ball, image_add_left, singleton_add, sub_neg_eq_add, add_comm y x]
+variables {ε δ : ℝ} {s : set E}
 
-@[simp] lemma ball_add_singleton (x y : E) (r : ℝ) :
-  ball x r + {y} = ball (x + y) r :=
-by simp [add_comm _ {y}, add_comm y]
+lemma ennreal.of_real_of_nonpos {r : ℝ} (hr : r ≤ 0) : ennreal.of_real r = 0 :=
+congr_arg coe $ real.to_nnreal_of_nonpos hr
```

Comment:
```suggestion
ennreal.of_real_eq_zero.mpr hr
```

This lemma is one direction of `ennreal.of_real_eq_zero`, but perhaps worth having because of the easily searchable name?

Reply:
Done in #13380